helm/dex-app/values.yaml

```diff
@@ -59,6 +59,7 @@ oidc:
     connectorName: ""
     connectorConfig: ""
     write_all_group: ""
+    connectors:
```

Comment:
On default this is empty. If populated it would look something like this, so basically exactly like what we have before but as a list and including an `id` field.
```
connectors:
- id: customer
  connectorType: github
  connectorName: customerconnector
  connectorConfig: |-
        clientID: 12345
        clientSecret: abcde
        loadAllGroups: false
        orgs:
        - name: myorg
          teams:
          - "Employees"
        redirectURI: someurl.com
  write_all_group: ""
- id: anothercustomer
  connectorType: github
  connectorName: customerconnector
  connectorConfig: |-
        clientID: 12345
        clientSecret: abcde
        loadAllGroups: false
        orgs:
        - name: myorg
          teams:
          - "Employees"
        redirectURI: someurl.com
  write_all_group: ""

```